docs/QuickStart.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_position: 3
+---
+
+# Quick Start Guide
+
+## 0. Prerequisites
+
+### For development
+
+This guide assumes you have completed the [Environment Setup](https://reactnative.dev/docs/set-up-your-environment) from the React Native docs. You'll also need the following:
+
+- [VS Code](https://code.visualstudio.com/) (or another IDE)
+- A macOS, Linux, or Windows (PowerShell or WSL2) with a terminal window open
+
+Keep in mind if you're looking to build iOS applications, you'll have to do this on macOS. Android can be developed from any OS.
+
+### For building
+
+You'll need:
+
+- EAS CLI installed
+
+```bash
+npm install -g eas-cli
+```
+
+- An Expo account, which you can [sign up for here](https://expo.dev/signup)
```

Comment:
Maybe denote this as "optional"?

Reply:
I would at least indicate that it's free to sign up

Reply:
So "An Expo account. [Create one for free here[(https://expo.dev/signup).